conda_forge_tick/mamba_solver.py

```diff
@@ -539,9 +488,19 @@ def clean():
     return repodata.channel_url
 
 
+def _func(feedstock_dir, additional_channels, conn):
+    res = _is_recipe_solvable(
+        feedstock_dir,
+        additional_channels=additional_channels,
+    )
+    conn.send(res)
+    conn.close()
```

Comment:
This needs to catch exceptions and pass them back to the parent:
```suggestion
    try:
        res = _is_recipe_solvable(
            feedstock_dir,
            additional_channels=additional_channels,
        )
        conn.send(res)
    except Exception as e:
        conn.send(e)
    finally:
        conn.close()
```